mava/networks/torsos.py

```diff
@@ -59,7 +59,9 @@ def setup(self) -> None:
     def __call__(self, observation: chex.Array) -> chex.Array:
         """Forward pass."""
         x = observation
-        for channel, kernel, stride in zip(self.channel_sizes, self.kernel_sizes, self.strides):
+        for channel, kernel, stride in zip(
+            self.channel_sizes, self.kernel_sizes, self.strides, strict=False
+        ):
```

Comment:
The default is `False `but maybe it's good to set it to `True` to avoid mismatch?
```suggestion
        for channel, kernel, stride in zip(
            self.channel_sizes, self.kernel_sizes, self.strides, strict=True
        ):
```

Reply:
This is a weird one, `ruff` just added this for me during pre-commit, so I think it's one of it's rules :shrug: 

I think true is good if we're having the keyword there anyways